### PR TITLE
Alignment of buttons

### PR DESCRIPTION
**Pull request by satcasm**
Issue Reference: #407 
Before:
image
After:
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "method-override": "^3.0.0",
     "moment": "^2.29.1",
     "mongoose": "^5.12.2",
-    "moment": "^2.29.1",
```
Comment (Pranav016) [inline package.json:28]: This package was removed ?

File: views/home.ejs [hunk 2]
```diff
@@ -67,9 +67,10 @@
     <% } else { %>
         <div class="orderForms row">
             <form action="/page/<%= current %>/<%= perPage %>" method="GET" class="quantityForm col-sm-6 text-center">
-                <label for="quantity">Number of posts:</label>
+                <label for="quantity" style="font-weight: bold;font-size: 1.4rem;">Number of posts:</label>
```
Comment (Pranav016) [inline views/home.ejs:70]: use class, inline style doesn't look good and the code isn't very clean that way.
From now onwards we are trying to maintain a good code base

File: views/home.ejs [hunk 3]
```diff
@@ -83,7 +84,8 @@
                         <option value="old one first">Old one first</option>
                     <% }  %>
                 </select>
-                <button class="button  arrow" type="submit" style="min-width: 10em;"> Go</button>
+                <button class="button  arrow" type="submit" style="min-width: 10em; padding-bottom: 0.2rem;
```
Comment (Pranav016) [inline views/home.ejs:87]: inline style ❌

File: views/home.ejs [hunk 4]
```diff
@@ -67,9 +67,10 @@
     <% } else { %>
         <div class="orderForms row">
             <form action="/page/<%= current %>/<%= perPage %>" method="GET" class="quantityForm col-sm-6 text-center">
-                <label for="quantity">Number of posts:</label>
+                <label for="quantity" style="font-weight: bold;font-size: 1.4rem;">Number of posts:</label>
                 <input type="number" name="perPage" min="1" max="10" value=<%= perPage %>>
-                <button class="button  arrow" type="submit" style="min-width: 10em;" >Go</button>
+                <button class="button  arrow" type="submit" style="min-width: 10em; padding-bottom: 0.2rem;
```
Comment (Pranav016) [inline views/home.ejs:72]: inline style ❌

**Review by Pranav016 — CHANGES_REQUESTED** [review]

**Comment by Pranav016** [comment]
Can you add a screenshot demonstrating the changes..

**Comment by Pranav016** [comment]
/gssoc/

File: package.json (hunk 1 above)
Comment (satcasm) [reply package.json:28]: No, it was causing some error so I to install it again.
So the previous one was deleted and this new one was added.

File: views/home.ejs (hunk 2 above)
Comment (satcasm) [reply views/home.ejs:70]: Okay, sure.

**Comment by satcasm** [comment]
Hey @Pranav016 , can you tell me why is it increasing the size of ```package-lock.json``` file when I am installing the packages again.
(I had to install the packages again as there were some issues while testing the site.)

**Comment by Pranav016** [comment]
> Hey @Pranav016 , can you tell me why is it increasing the size of `package-lock.json` file when I am installing the packages again and also how to resolve it as it adds around `3000` lines ?
> (I had to install the packages again as there were some issues while testing the site.)

No idea, I would suggest pull the changes again in a few mins and try again cause there are some conflicts as well.

**Comment by Pranav016** [comment]
Closing this PR because the no. of commits required to resolve have exceeded the complexity of the issue

**Pull request closed without merging**